config/environments/test.rb

```diff
@@ -6,6 +6,8 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.default_url_options = { host: ENV['HOSTNAME'] || 'localhost:3000' }
```

Comment:
I did this in #75 so we may have a merge conflict depending on which gets merged first. Or this line may be in there twice if we put them at different parts of the file.